blueocean-dashboard/src/main/js/components/ColumnFilter.jsx

```diff
@@ -26,7 +26,11 @@ export class ColumnFilter extends Component {
         if (event.type === 'select'
             || event.type === 'blur'
             || (event.type === 'keypress' && event.key === 'Enter')) {
+            this.setState({ originalValue: value });
             onChange(value);
+            setTimeout(() => {
+                this.refs.autocomplete.refs.input.blur();
```

Comment:
Yes, but this line can't be refactored to use callback refs without also changing the autocomplete component, so we should probably leave it until we can replace it with our dropdown.